tests/testthat/test-warning.R

```diff
@@ -100,8 +100,8 @@ test_that("warning raised if outputs exceed threshold", {
   expect_length(out$warnings, 2)
   expect_equal(
     out$warnings[[1]]$text,
-    "HIV prevalence is higher than 40% for some age/sex disaggregates")
+    "HIV prevalence is higher than 40% for:  CY2016Q1 MWI_1_1_demo both Y000_004")
```

Comment:
Do we need to put keys in for this message too? Or will make sense to people without?

Reply:
There are so many possible cal quarter/age/sex combinations, I don't think keys will be possible. @jeffeaton do you think read out is clear enough here?